package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "pokemon-showdown-login",
+  "version": "0.1.0",
+  "homepage": "https://pokemonshowdown.com",
+  "license": "AGPL-3.0",
+  "scripts": {
+    "lint": "eslint . --cache --ext .js,.ts",
+    "build": "npx tsc",
+    "start": "npm run build && npx pm2 start .dist"
```

Comment:
For actual production usage, I doubt we'll use a workflow like this... I don't have a concrete suggestion at the moment but just noting for the record.